Eduria/Eduria/wwwroot/js/site.js

```diff
@@ -1,15 +1,76 @@
-﻿// Flexslider
```

Comment:
In alle javascript moeten de functies voorzien zijn van comments die aangeven wat de functie doet en een beschrijving van de gegeven parameters.